.github/workflows/operate-e2e-tests.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           - 9200:9200
           - 9300:9300
       zeebe:
-        image: camunda/zeebe:8.5.0-alpha2
```

Comment:
💭 @pedesen would it actually make sense to use snapshot here? 🤔

Reply:
@Zelldon I'm not sure about it. In case Zeebe merges a change which requires an adjustment in Operate, the test might fail until it is fixed in Operate. On the other hand we might be aware of these problems earlier if we use snapshot.

Reply:
I'll discuss with team next week.